api/index.js

```diff
@@ -136,6 +136,16 @@ app.get('/customers/:id/documents', async (req, res) => {
       getSystemId
     }
   );
+
+  const UHWFetchDocumentsGateway = require('./lib/gateways/UHW/UHWFetchDocuments')(
```

Comment:
lets keep all our casing the same (uhwFetch...)

Reply:
I'll fix that in next commit, after shifting it to libDependencies, cause some of the consts in libdependencies are uppercase too. Is that cool?